apis/spark/src/main/java/io/tiledb/vcf/VCFInputPartitionReader.java

```diff
@@ -304,6 +304,12 @@ private void initVCFReader() {
       if (bedURI.isPresent()) {
         vcfReader.setBedFile(bedURI.get().toString());
       }
+
+      // Set BED array
+      Optional<URI> bedArrayURI = options.getBedArrayURI();
+      if (bedArrayURI.isPresent()) {
+        vcfReader.setBedArray(bedArrayURI.get().toString());
```

Comment:
We recently removed `vcfReader.setBedArray` to simplify the API. Please use `vcfReader.setBedFile` instead. I believe that's why CI is failing.